combinations/hash.tsv

```diff
@@ -392,3 +392,4 @@ bioconductor-clusterprofiler=4.6.0,bioconductor-org.hs.eg.db=3.16.0
 xopen=1.1.0,pysam=0.16.0
 bioconductor-purecn=2.4.0,bioconductor-txdb.hsapiens.ucsc.hg38.knowngene=3.16.0,bioconductor-org.hs.eg.db=3.16.0,bioconductor-txdb.hsapiens.ucsc.hg19.knowngene=3.2.2
 python=3.6.13,pandas=1.1.5,numpy=1.19.5,jinja2==2.11.3,pyyaml=5.3,fire=0.3
+python=2.7,pysam=0.20.0,flask=1.1.2,numpy=1.16.5,matplotlib=2.2.4,scipy=1.2.0,mosek::mosek=8.0.60,intervaltree=3.0.2,future=0.18.2
```

Comment:
python2? Its not supported anymore, should we really create container for it?

Reply:
this is the only way the software dependencies were working for PrepareAA, because it uses mosek 8 and multiple conflicts arose. I tried many combinations, but only the versions for python2 worked

Reply:
I'll see if I find another solution

Reply:
yeah no, there does not seem to be another solution, sorry. Would you be willing to merge ?